massgen/frontend/displays/textual_terminal_display.py

```
@@ -456,6 +456,19 @@ def _build_subagent_display_data(
+    # Recalculate log_path when the server assigned a different ID than the
+    # placeholder used at card creation (e.g. "subagent_0" → "subagent_1").
+    # The stale log_path still points to the old ID's directory.
+    if log_path and existing and subagent_id != existing.id and not sa_data.get("log_path"):
+        try:
+            from massgen.logger_config import get_log_session_dir
+
+            session_dir = get_log_session_dir()
+            if session_dir:
+                log_path = str(session_dir / "subagents" / subagent_id)
+        except Exception:
+            pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)